src/main/webapp/app/entities/exam-user.model.ts

```diff
@@ -10,10 +10,10 @@ export class ExamUser implements BaseEntity {
     public plannedSeat?: string;
     public signingImagePath?: string;
     public studentImagePath?: string;
-    public didCheckImage?: boolean;
-    public didCheckName?: boolean;
-    public didCheckLogin?: boolean;
-    public didCheckRegistrationNumber?: boolean;
+    public didCheckImage = false; // default value
```

Comment:
I really dislike having `// default value` comments because it implies that initializing variables with that value where could not be a default value.